src/main/java/digital/slovensko/autogram/server/dto/ServerSigningParameters.java

```diff
@@ -196,6 +198,11 @@ private ASiCContainerType getContainer() {
         return container;
     }
 
+    private boolean shouldAutoLoadEform() {
+        // justice.gov.sk Forms need to be auto-loaded to get the embedded xsd and xslt
```

Comment:
Tejto logike uplne nerozumiem. Nie je to rovnake ako pri inych forms? Proste ked neposles autoload a neposles ani xslt/schemu tak to vyhubuje?

Reply:
Toto je výnimka kvôli podpisovaniu na slovensko.sk. Dáš podpísaný návrh na zmenu do prílohy a chceš pridať podpis ďalšieho človeka. Slovensko.sk to nijako zjavne nerieši, tak ti to pošle rovnako ako iné formuláre, ale s prázdnymi xsd a xslt.

Je chyba, že extrakcia xsd a xslt z xdc prebieha v autoLoadEform v `EFormResources.java`, pričom sa rozhoduje podľa identifikátora formulára a nie podľa toho, či v tom xdc už sú embedded alebo referenced schémy. Táto extrakcia by mala prebehnúť v `SigningParameters::buildParameters` ešte pred autoLoadEform. Toto upravím. Tým pádom jediná výnimka bude pre embedUsedSchemas, ktoré sa overridne na true vždy, ak je to ORSR formulár.

Reply:
Ok, trochu som to prerobil. Ak je to XDC (čo sa po novom zisťuje iba raz!), najprv sa skúsi, či to nie je embedded, potom sa ide normálne ďalej.